include/common/configure.h

```diff
@@ -221,6 +221,18 @@ class Configure {
 
   void removeProposal(const Proposal Type) noexcept {
     std::unique_lock Lock(Mutex);
+    if (Type == Proposal::FunctionReferences &&
+        Proposals.test(static_cast<uint8_t>(Proposal::GC))) {
+      // Proposal dependency: GC depends FunctionReferences.
```

Comment:
In my testing, it seems like the `enable-gc` flag implies `enable-function-references` automatically. Is that the case? If so, might be good to expand on the comment here (e.g. should be impossible, disable code coverage)

Reply:
Yes, the `enable-gc` flag implies `enable-function-references` automatically.
For the code coverage, it will be improved after applying the GC spec test suites for interpreter.
We turn off this now due to failing the several tests, and will fix that soon.